### PR TITLE
RHDEVDOCS-2900 Tracker for #30976 Issue in file logging/cluster-loggi…

### PR DESCRIPTION
**Pull request by rolfedh**
…ng-deploying.adoc Add "OpenShift" to "Elasticsearch Operator"

- For versions 4.7+
- https://issues.redhat.com/browse/RHDEVDOCS-2900
- Direct link to doc preview: https://deploy-preview-31302--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging.html
- Ready for SME, QE, and peer review
- Please merge when ready
- Please close the original issue: https://github.com/openshift/openshift-docs/issues/30976

### CODE REVIEW

**Review by JStickler — APPROVED** [review]
LGTM (since I work on the Jaeger and Service Mesh content).

Also, it looks like this PR also addresses https://github.com/openshift/openshift-docs/issues/30981.

File: modules/cluster-logging-release-notes-5.0.0.adoc [hunk 1]
```diff
@@ -19,23 +19,23 @@ With this release, Cluster Logging becomes Red Hat OpenShift Logging, version 5.
...
+This release updates the display name of the OpenShift Elasticsearch Operator and operator maturity level. The new display name and clarified specific use for the OpenShift Elasticsearch Operator are updated in Operator Hub.
 
 [discrete]
 [id="openshift-logging-5-0-es-csv-success"]
 // https://bugzilla.redhat.com/show_bug.cgi?id=1913464
-=== Elasticsearch Operator reports on CSV success
+=== OpenShift Elasticsearch Operator reports on CSV success
 
-This release adds reporting metrics to indicate that installing or upgrading the Elasticsearch Operator ClusterServiceVersion (CSV) was successful. Previously, there was no way to determine, or generate an alert, if the CSV installation or upgrade for the Elasticsearch Operator failed. Now, an alert is provided as part of the Elasticsearch Operator.
+This release adds reporting metrics to indicate that installing or upgrading the OpenShift Elasticsearch Operator ClusterServiceVersion (CSV) was successful. Previously, there was no way to determine, or generate an alert, if the CSV installation or upgrade for the OpenShift Elasticsearch Operator failed. Now, an alert is provided as part of the OpenShift Elasticsearch Operator.
```
Comment (jeana-redhat) [inline modules/cluster-logging-release-notes-5.0.0.adoc:38]: Probably out of scope for this, but just noting that ClusterServiceVersion is a custom resource and should probably be written as "`ClusterServiceVersion` object (CSV)" or "`ClusterServiceVersion` custom resource (CSV)" according to standards. I see this one is used very inconsistently in the repo, though.

File: modules/cluster-logging-release-notes-5.0.0.adoc [hunk 2]
```diff
@@ -153,23 +153,23 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
...
+* An earlier bug fix to address issues related to certificate generation introduced an error. Trying to read the certificates caused them to be regenerated because they were recognized as missing. This, in turn, triggered the OpenShift Elasticsearch Operator to perform a rolling upgrade on the Elasticsearch cluster and, potentially, to have mismatched certificates. This bug was caused by the operator incorrectly writing certificates to the working directory. The current release fixes this issue. Now the operator consistently reads and writes certificates to the same working directory, and the certificates are only regenerated if needed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905910[*BZ#1905910*])
```
Comment (jeana-redhat) [inline modules/cluster-logging-release-notes-5.0.0.adoc:166]: There's a couple other lowercase "operator" instances here to capitalize

File: modules/dedicated-cluster-install-deploy.adoc [hunk 3]
```diff
@@ -4,18 +4,18 @@
 
 [id="dedicated-cluster-install-deploy"]
 
-= Installing OpenShift Logging and Elasticsearch Operators
+= Installing OpenShift Logging and OpenShift Elasticsearch Operators
 
 You can use the {product-title} console to install OpenShift Logging by deploying instances of
-the OpenShift Logging and Elasticsearch Operators.  The Cluster Logging Operator
-creates and manages the components of the logging stack.  The Elasticsearch Operator
+the OpenShift Logging and OpenShift Elasticsearch Operators.  The Cluster Logging Operator
```
Comment (jeana-redhat) [inline modules/dedicated-cluster-install-deploy.adoc:10]: Maybe replace the instances of `.  ` (two spaces) here with `. ` (one space)?

File: modules/jaeger-config-storage.adoc [hunk 4]
```diff
@@ -73,10 +73,10 @@ When the `storage:type` is set to `elasticsearch` but there is no value set for
 
 [NOTE]
 ====
-If you already have installed Elasticsearch as part of OpenShift logging, the Jaeger Operator can use the installed Elasticsearch Operator to provision storage.
+If you already have installed Elasticsearch as part of OpenShift logging, the Jaeger Operator can use the installed OpenShift Elasticsearch Operator to provision storage.
 ====
 
-The following configuration parameters are for a _self-provisioned_ Elasticsearch instance, that is an instance created by the Jaeger Operator using the Elasticsearch Operator.  You specify configuration options for self-provisioned Elasticsearch under `spec:storage:elasticsearch`  in your configuration file.
+The following configuration parameters are for a _self-provisioned_ Elasticsearch instance, that is an instance created by the Jaeger Operator using the OpenShift Elasticsearch Operator.  You specify configuration options for self-provisioned Elasticsearch under `spec:storage:elasticsearch`  in your configuration file.
```
Comment (jeana-redhat) [inline modules/jaeger-config-storage.adoc:79]: `.  ` here as well

File: modules/jaeger-install-elasticsearch.adoc [hunk 5]
```diff
@@ -20,7 +20,7 @@ Do not install Community versions of the Operators. Community Operators are not
 
 [NOTE]
 ====
-If you have already installed the Elasticsearch Operator as part of OpenShift Logging, you do not need to install the Elasticsearch Operator again.  The Jaeger Operator will create the Elasticsearch instance using the installed Elasticsearch Operator.
+If you have already installed the OpenShift Elasticsearch Operator as part of OpenShift Logging, you do not need to install the OpenShift Elasticsearch Operator again.  The Jaeger Operator will create the Elasticsearch instance using the installed OpenShift Elasticsearch Operator.
```
Comment (jeana-redhat) [inline modules/jaeger-install-elasticsearch.adoc:23]: `.  ` here as well

File: modules/jaeger-rn-fixed-issues.adoc [hunk 6]
```diff
@@ -17,7 +17,7 @@ Result - How has the behavior changed as a result?  Try to avoid “It is fixed
...
 
-* link:https://issues.jboss.org/browse/TRACING-1631[TRACING-1631] Multiple Jaeger production instances, using same name but within different namespaces, causing Elasticsearch certificate issue.  When multiple service meshes were installed, all of the Jaeger Elasticsearch instances had the same Elasticsearch secret instead of individual secrets, which prevented the Elasticsearch Operator from communicating with all of the Elasticsearch clusters.
+* link:https://issues.jboss.org/browse/TRACING-1631[TRACING-1631] Multiple Jaeger production instances, using same name but within different namespaces, causing Elasticsearch certificate issue.  When multiple service meshes were installed, all of the Jaeger Elasticsearch instances had the same Elasticsearch secret instead of individual secrets, which prevented the OpenShift Elasticsearch Operator from communicating with all of the Elasticsearch clusters.
```
Comment (jeana-redhat) [inline modules/jaeger-rn-fixed-issues.adoc:20]: `.  ` here as well

**Review by jeana-redhat — APPROVED** [review]
LGTM - noted a few small things that you may or may not consider in scope for this (not the change you were looking at, but in the same sentences). LMK if you want to do anything about them or if you'd like me to go ahead and merge and CP to 4.7+

File: modules/cluster-logging-release-notes-5.0.0.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.0.adoc:38]: Thanks. Updated.

File: modules/cluster-logging-release-notes-5.0.0.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.0.adoc:166]: Thanks. I think I caught and fixed these.

File: modules/jaeger-config-storage.adoc (hunk 4 above)
Comment (rolfedh) [reply modules/jaeger-config-storage.adoc:79]: Updated this one plus about sixteen others in this file. Thanks.

File: modules/jaeger-install-elasticsearch.adoc (hunk 5 above)
Comment (rolfedh) [reply modules/jaeger-install-elasticsearch.adoc:23]: Replaced this one plus three others in this file.

File: modules/dedicated-cluster-install-deploy.adoc (hunk 3 above)
Comment (rolfedh) [reply modules/dedicated-cluster-install-deploy.adoc:10]: Fixed two in this file.

**Pull request merged**